### DISCUSSION TITLE
分布式训练卡住

### DISCUSSION THREAD

**Discussion by zjlw** [PaddlePaddle/PaddleOCR#14148] [Q&A]
环境4张l40s
官方镜像源：registry.baidubce.com/paddlepaddle/paddle:latest-dev-cuda12.0-cudnn8.9-trt8.6-gcc12.2
paddle版本:2.6.1
运行命令nohup python -m paddle.distributed.launch --gpus "01,2,3" python tools/train.py -c configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_teacher.yml > /dev/null   2>&1   &

显卡使用率100%，但显存占用基本上没有，一直卡住

**Comment by UserWangZz** [comment]
--gpus "01,2,3"，0和1之间没有逗号，请问在执行的时候是一致的吗？

**Reply by zjlw** [reply]
实际上是有逗号的，
